### PR TITLE
style login page [after #44]

### PR DESCRIPTION
**Pull request by bcuz**
Easiest way to review this is to look at my individual commits. This PR builds on top of Wilfreds open #44

### CODE REVIEW

File: socialtours/src/Auth.js [hunk 1]
```diff
@@ -28,11 +28,13 @@ export default class Auth {
 		});
 	}
 
-	googleLogin() {
+	googleLogin = e => {
+		e.preventDefault();
```
Comment (GannonDetroit) [inline socialtours/src/Auth.js]: I'm glad you added this.

File: socialtours/src/components/DesignComponents/GlobalStyle.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
 import { createGlobalStyle } from "styled-components";
 import "typeface-roboto";
 
-//import { colors, color, fontStyles, fontSizing } from "./theme";
+import { fontStyles } from "./theme";
 import "../../reset.css";
 
 const GlobalStyle = createGlobalStyle`
+@import url('https://use.fontawesome.com/releases/v5.0.13/css/all.css');
```
Comment (GannonDetroit) [inline socialtours/src/components/DesignComponents/GlobalStyle.js]: good job helping to keep design consistent

File: socialtours/src/components/UserComponents/FormStyles.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
+import styled, { css } from "styled-components";
```
Comment (GannonDetroit) [inline socialtours/src/components/UserComponents/FormStyles.js]: Love this. excellent job

**Comment by bcuz** [comment]
[style login](https://trello.com/c/TInc1RE4/84-style-login)

**Pull request merged**